fineract-provider/src/main/java/org/apache/fineract/organisation/office/data/OfficeData.java

```diff
@@ -40,6 +42,34 @@
     @SuppressWarnings("unused")
     private final Collection<OfficeData> allowedParents;
 
+    //import fields
+    private transient Integer rowIndex;
+    private String locale;
+    private String dateFormat;
+
+    public OfficeData(String name, Long parentId,LocalDate openingDate,String externalId,Integer rowIndex) {
+
```

Comment:
@wkk91193 Hey, please follow convention here. All instantiator functions are static methods which should in turn call the private constructor.